src/main/src/main/java/org/geoserver/GeoServerConfigurationLock.java

```diff
@@ -38,7 +38,7 @@ public class GeoServerConfigurationLock {
      */
     public static long DEFAULT_TRY_LOCK_TIMEOUT_MS = 30000;
 
-    private static final Level LEVEL = Level.FINE;
+    private static final Level LEVEL = Level.INFO;
```

Comment:
Do we really want configuration lock at "info" level? Assume you had this on for troubleshooting; looks like LEVEL just controls one log message; so setting this as FINE and using log4j.xml logging of locking should be good enough.

Reply:
Yep, it was for troubleshooting and forgot to back it out. NIce catch, fixing it.